src/sickle.ts

```diff
@@ -241,6 +246,14 @@ class Annotator {
       this.emit(jsDocTag);
       this.emit(' {');
     }
+    this.visitType(type);
+    if (jsDocTag) {
+      this.emit('}');
+    }
+    this.emit(' */');
+  }
+
+  private visitType(type: ts.TypeNode, optional?: boolean) {
```

Comment:
The difference between `maybeVisitType` and `visitType` is not very clear from the function name or implementation. One emits jsDoc annotations and the other doesn't. The mega-loop 'visit' calls into both, so we can't argue that one is an inner helper.

Maybe some renaming would help?